README.md

```diff
@@ -297,6 +297,12 @@ While developing, you will probably rely mostly on `yarn watch`; however, there
   yarn start
   ```
 
+### Building standalone mobile apps for Play Store and App Store
+1. Compile project for production with `ios` and `android` set to `true` in `app.json`.
+2. Run `yarn exp ba` to launch building signed `.apk` or `yarn exp bi` for signed `.iap`.
```

Comment:
Maybe it's more clear `Compile project `yarn build` for production with `ios` and `android` set to `true` in `app.json`.`